### ISSUE TITLE
[ARCHIVE] Brigade

### ISSUE THREAD

**Issue by amye** [cncf/toc#929]
- [x] TOC vote to archive passed
- [x] Move logo to archived on CNCF.io	
- [x] Remove from landscape		
- [x] Archive maintainers mailing list	
- [x] Remove ML from maintainers@ mailing group
- [x] Remove/Archive on maintainers.cncf.io
- [x] Remove logo from all slide decks	
- [x] Remove logo from event sites	
- [x] Move logo to archived in artwork	
- [x] Archive DevStats (or archive?)	
- [x] Notify Speakers team to remove from CFPs	
- [x] Remove from service desk or archive	
- [x] Remove from CNCF store

**Comment by amye** [comment]
https://github.com/cncf/artwork/pull/360

**Comment by amye** [comment]
https://github.com/cncf/landscape/pull/2764

**Comment by caniszczyk** [comment]
Before we close this, we need to ensure the website has an opentracing.io style "Archived" banner

Patrice will work on this

**Comment by chalin** [comment]
> Before we close this, we need to ensure the website has an opentracing.io style "Archived" banner
> 
> Patrice will work on this

Done. A banner has been added to all sections of the website:

- https://brigade.sh/
- https://github.com/brigadecore/brigade/issues/1983

Note that I chose _not_ to add a banner to the (old) v1 docs. There is a banner in the v1 docs that refers the reader to the (latest) v2 docs, which themselves have a banner saying that Brigade is archived. That seemed enough. Let me know if that works for you, or if you'd like anything else done.
